More predicates

csierra opened the pull request.

Commit:
initial version

adolfopa committed:
bundle(Atom) [working] and bundle(Var) [broken]

Commit:
Bundle predicate

Commit:
WIP

Commit:
added java 8 compile support

Commit:
More predicates

Services are matched against service.id.

liferay-continuous-integration commented:
<h1>Build Started</h1><h4>Job Link:</h4><p><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest(master)/52/">test-portal-acceptance-pullrequest(master)</a></p>

csierra commented:
ci:stop

liferay-continuous-integration commented:
<h1>Some tests FAILED!</h1><p>Build Time: 2 minutes 22 seconds</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br />Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/10c8b0db98be035c6b806133a4c752841e22e5cd">10c8b0db98be035c6b806133a4c752841e22e5cd</a></p><h4>Job Summary:</h4><ul><li><strike><strong><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest(master)/52/">test-portal-acceptance-pullrequest(master)</a></strong></strike></li><li><strike><strong><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/2515/">test-portal-acceptance-pullrequest-batch(master)/compile-jsp-jdk7</a></strong></strike></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-source(master)/70/">test-portal-acceptance-pullrequest-source(master)</a></li></ul><h5>For more details click <a href="https://test-1-15.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/52/jenkins-report.html">here</a>.</h5><hr /><h4>Failed Jobs:</h4><ol><li><h5><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest(master)/52/">test-portal-acceptance-pullrequest(master)</a></h5><h6>Job Results:</h6><p>1 Job Passed.<br />2 Jobs Failed.</p><pre><code>Completed with the status of ABORTED.</code></pre></li><li><h5 job-result=""><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/2515/">test-portal-acceptance-pullrequest-batch(master)/compile-jsp-jdk7</a></h5></li></ol>

Commit:
Much better now

This works now:
```ins:query "service(B, S), interfaces(S, I), member('com.liferay.application.list.PanelApp', I)."```

and returns the bundles that export a service that implements 'com.liferay.application.list.PanelApp'.

adolfopa commented:
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest(master)/498/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-6.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/498/jenkins-report.html">here</a>.</b></p>

adolfopa commented:
ci:stop

liferay-continuous-integration commented:
<h1>Some tests FAILED.</h1><p>Build Time: 15 minutes 12 seconds</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br />Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/355a534fb71403a9413e9c505b22bced7a5190ca">355a534fb71403a9413e9c505b22bced7a5190ca</a></p><h4>Job Summary:</h4><ul><li><strike><strong><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest(master)/498/">test-portal-acceptance-pullrequest(master)</a></strong></strike></li><li><strike><strong><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-dist(master)/296/">test-portal-acceptance-pullrequest-dist(master)/jdk8</a></strong></strike></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-source(master)/811/">test-portal-acceptance-pullrequest-source(master)</a></li></ul><h5>For more details click <a href="https://test-1-6.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/498/jenkins-report.html">here</a>.</h5><hr /><h4>Failed Jobs:</h4><ol><li><h5><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest(master)/498/">test-portal-acceptance-pullrequest(master)</a></h5><h6>Job Results:</h6><p>1 Job Passed.<br />2 Jobs Failed.</p><pre><code>Completed with the status of ABORTED.</code></pre></li><li><h5 job-result=""><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-dist(master)/296/">test-portal-acceptance-pullrequest-dist(master)/jdk8</a></h5></li></ol><h5>For upstream results, click <a href="http://test-1-1/job/test-portal-acceptance-upstream(master)/">here</a>.</h5>

adolfopa commented:
Pull request submitted to https://github.com/liferay-lima/liferay-portal/pull/272. [See changes here.](https://github.com/adolfopa/liferay-portal/compare/327807e9cadf~1...327807e9cadf)

adolfopa commented:
:zombie_woman:
